src/io/file.js

```diff
@@ -17,9 +17,13 @@ module.exports = function File(options, processor) {
     lnk.download = (new Date()).toISOString().replace(/:/g, "_") + "." + format;
     // create a "fake" click-event to trigger the download
     if (document.createEvent) {
-      event = document.createEvent("MouseEvents");
-      event.initMouseEvent("click", true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
-      lnk.dispatchEvent(event);
+      const mouseEvent = getMouseEvent();
```

Comment:
This looks super. Perfectly done with regard to src and dist - thank you!!!! However, I'm now seeing an error when I try to download:

```
Uncaught ReferenceError: getMouseEvent is not defined
    at Object.downloadImage (infragram.js? [sm]:595:18)
    at Object.download (infragram.js? [sm]:110:18)
    at HTMLButtonElement.onclick ((index):163:107)
```

Do we need to do something closer to the usage described in https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/MouseEvent ? Or can you link to where you found the solution you used here... maybe there is some typo? 

I'm sure we'll solve it and as soon as we do, we can `grunt build` again and test it out. Thank you @Somya-Singhal !!

Reply:
Here is the link where I found the solution: https://github.com/naver/billboard.js/pull/252/commits/dd3b0e4b662624438d12dc99e00bdac9886e33bc

I think I have to define getMouseEvent() function as specified in this link on line no. 16, that will fix this issue.

Reply:
Hello @jywarren, I have checked after defining getMouseEvent() function it's now working. I have attached one screenshot below to show that the image is now getting downloaded. 
<img width="960" alt="image" src="https://user-images.githubusercontent.com/88088520/163122299-b424fd66-df3b-4c00-ae95-d9b5f19adbf1.png">

Please can you now review it and suggest if any changes are required?

Reply:
Hello @TildaDares @jywarren, Please can you review this pull request?

Reply:
Hello @jywarren, is there anything I had mistaken in this PR that needs to be changed? I would like to hear your feedback!🙂